From 1ff3db7f7458f852ab241c5911e396a907e7478e Mon Sep 17 00:00:00 2001
From: jerome <jerome.anand@intel.com>
Date: Tue, 12 Mar 2013 20:45:44 +0530
Subject: [PORT FROM MAIN][DO NOT UPSTREAM][Remote BGM] Support playback control using new flag

BZ: 92431

The audio/video playback is started based on the set
parameters by the customized player and using the same
in audio player for starting BGM

Change-Id: I49b85553d195277f5cde9ef2ff9591a0765162bc
Orig-Change-Id: I689a11cb3d8ba002d13e3f05cca4435b1714d5cc
Signed-off-by: jerome <jerome.anand@intel.com>
---
 include/system/audio.h |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/include/system/audio.h b/include/system/audio.h
index 6622b83..60ce14d 100644
--- a/include/system/audio.h
+++ b/include/system/audio.h
@@ -397,7 +397,8 @@ typedef enum {
     AUDIO_OUTPUT_FLAG_FAST = 0x4,       // output supports "fast tracks",
                                         // defined elsewhere
     AUDIO_OUTPUT_FLAG_DEEP_BUFFER = 0x8, // use deep audio buffers
-    AUDIO_OUTPUT_FLAG_COMPRESS_OFFLOAD = 0x10 // Use compress offload in DOT
+    AUDIO_OUTPUT_FLAG_COMPRESS_OFFLOAD = 0x10, // Use compress offload in DOT
+    AUDIO_OUTPUT_FLAG_REMOTE_BGM = 0x20 // used for remote background playback
 } audio_output_flags_t;
 
 static inline bool audio_is_output_device(audio_devices_t device)
-- 
1.7.4.1

